workflows/offline_ml_diags/offline_ml_diags/compute_diags.py

```diff
@@ -415,9 +415,12 @@ def main(args):
     ds_diagnostics["time"] = times_used
     ds_diurnal["time"] = times_used
 
-    # save jacobian
+    # save model senstivity figures: jacobian (TODO: RF feature sensitivity)
     try:
-        plot_jacobian(model, args.output_path)  # type: ignore
+        plot_jacobian(
```

Comment:
not new to this PR, so this is just a clarification question, but why does this plot get created as part of "compute diags" instead of as part of the report generation like everything else? I was confused for a while since `render_model_sensitivity` wasn't actually generating any figures.

Reply:
Since the jacobian calculation is part of the keras model, doing this here avoids having to read in the model in the `create_report` step.

Reply:
right, I guess what I mean is, shouldn't "compute diags" calculate the Jacobian and then save it to one of the netCDFs to then get loaded/plotted later?